packages/react-components/src/components/CameraButton.tsx

```diff
@@ -150,7 +150,9 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
 
   let disabled = props.disabled || waitForCamera;
   /* @conditional-compile-remove(rooms) */
-  disabled = disabled || !_usePermissions().cameraButton;
+  const permissions = _usePermissions();
+  /* @conditional-compile-remove(rooms) */
+  disabled = disabled || !permissions.cameraButton;
```

Comment:
are these the same but we just want to give them more space?

Reply:
I believe the logical OR here is conditionally calling a react hook.